src/components/families/family-list/family-sidebar/family-sidebar-form/FamilySidebarForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from "react";
+
+import { Button, Typography } from "@material-ui/core";
+
+import { FamilyStudentRequest } from "api/types";
+import FamilyParentFields from "components/families/family-form/family-parent-fields";
+import StudentForm from "components/families/family-form/student-form";
+import {
+  FamilyFormData,
+  familyFormDataToFamilyRequest,
+} from "components/families/family-form/utils";
+import StudentRole from "constants/StudentRole";
+import { DynamicFieldsContext } from "context/DynamicFieldsContext";
+
+type Props = {
+  family: FamilyFormData;
+  isEditing: boolean;
+  onCancel: () => void;
+  onChange: (family: FamilyFormData) => void;
+  onSubmit: (family: FamilyStudentRequest) => void;
```

Comment:
at first FamilyStudentRequest was a little confusing, because it sounded like only updating the students in the family (I was expecting to see FamilyDetailRequest or something). But then when I looked into the object it found out it was family and related students (request).

Reply:
yeah naming this was hard, i think we have the same issue on the backend because the Family type corresponds to the Family model on the backend, but our request includes the family + its students

i'll try to think of a less confusing name for this